__tests__/app.test.js

```diff
@@ -80,4 +80,55 @@ describe("CORE: GET /api/articles/:article_id error test suite", () => {
             expect(body.msg).toBe("Not found.");
         })
     })
+});
+describe("CORE: GET /api/articles functionality test suite", () => {
+    test("returns a 200 GET request with an article array of article objects without a body property in date descending order", () => {   
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toHaveLength(13);
+            const isSorted = body.articles.every((val, i, arr) => !i || arr[i-1].created_at >= val.created_at);
```

Comment:
This is a very inventive line of code! If you wanted to explore a nicer way of handling this, the npm package jest sorted can do just this for you :)
https://www.npmjs.com/package/jest-sorted